File: ৫২টি প্রোগ্রামিং সমস্যা ও সমাধান/প্রোগ্রামিং সমস্যা ২ — জোড়-বিজোড় ২.py
T = int(input())  # Take number of test cases

for _ in range(T):
    n = input().strip()  # Read input as a string and remove extra spaces

    # Get the last character of the string
    last_char = n[-1]

    # Check if last digit is even or odd
    if last_char in '02468':
        print("even")
    else:
        print("odd")

File: ৫২টি প্রোগ্রামিং সমস্যা ও সমাধান/প্রোগ্রামিং সমস্যা ২ — জোড়-বিজোড় ২.c
#include <stdio.h>
#include <string.h>  // Include for strlen()

int main() {
    
    int T, i; 
    char n[101];  // String input
    
    scanf("%d", &T);  // Take number of test cases
    
    for(i = 1; i <= T; i++){
        scanf("%s", n);  // Read input as a string

        // Get last character of the string
        char last_char = n[strlen(n) - 1];

        // Check if last digit is even or odd
        if (last_char == '0' || last_char == '2' || last_char == '4' || 
            last_char == '6' || last_char == '8') {
            printf("even\n");
        } else {
            printf("odd\n");
        }
    }
    
    return 0;
}
